views/layout.hbs

```diff
@@ -2,7 +2,8 @@
 <html>
   <head>
     <title>{{title}}</title>
-    <link rel='stylesheet' href='/stylesheets/style.css' />
+    {{{ css 'application.css' }}}
```

Comment:
This line generates this text:
<link type='text/css' href='/assets/globals.css?body=1' rel='stylesheet' media='screen' name='css' hash='[object Object]' data='[object Object]'>
<link type='text/css' href='/assets/application.css?body=1' rel='stylesheet' media='screen' name='css' hash='[object Object]' data='[object Object]'>